.github/workflows/macos.yml

```
@@ -7,7 +7,7 @@ env:
   CI_NIX_SHELL: true
 
 jobs:
-  test_macos:
+  x86_64-macos-test:
```

I'd suggest to use `x86_64-darwin` like in the LLVM target triple:

```suggestion
  x86_64-darwin-test:
```